samples/SocialSample/Startup.cs

```diff
@@ -199,13 +206,10 @@ 127.0.0.1 MsSecSample.localhost.this
                         await context.Response.WriteAsync("<a href=\"?authscheme=" + type.AuthenticationScheme + "\">" + (type.Caption ?? "(suppressed)") + "</a><br>");
                     }
                     await context.Response.WriteAsync("</body></html>");
-                });
-            });
-
-            // Sign-out to remove the user cookie.
-            app.Map("/logout", signoutApp =>
-            {
-                signoutApp.Run(async context =>
+                    return;
+                }
+                
+                if (context.Request.Path.StartsWithSegments(new PathString("/logout")))
                 {
                     await context.Authentication.SignOutAsync(CookieAuthenticationDefaults.AuthenticationScheme);
                     context.Response.ContentType = "text/html";
```

Comment:
Yeah well, as you can tell, the samples code for security is basically a little test app (that I never ever run clearly)  I mostly rely on MVC/Identity/MusicStore for functional verification, but I should use this as a canary as well (until we get real functional tests...in the repo sigh)